client/components/repository/Workflow/Filters/index.vue

```diff
@@ -12,7 +12,9 @@
       @change="updateFilter('status', $event)"
       :items="statusOptions"
       :value="status"
+      item-text="label"
```

Comment:
Should we make this a default value for `StatusFilter`'s `ItemText` prop (along with `ItemValue`)?
If we change the `Status` model in the future, we'd need to change default value _only_ in the `StatusFilter` component.